Update nmeToSfe.md

alexchuber opened the pull request:
Clarified descriptions for SmartFilterTexture usage with ImageSource.

Also removed note about `Is Main Input` for now.

deltakosh commented on content/resources/Playground_aspects/External_PG_assets.md:
````diff
@@ -17,17 +17,24 @@ Sometimes you might want to load you own assets into the playground. Should the
+ replace '/commit/' with '@'
 
  ```javascript
 myGithubUserName/myRepository@2bd79648e08709145cd9575e6679b2ea360f12f6
````
same

flopanzer replied:
```suggestion
myGithubUserName/myRepository@hash
```

deltakosh replied:
Well my problem is that the entire line is not supposed to be in the .md file ;)

flopanzer commented on content/resources/Playground_aspects/External_PG_assets.md:
````diff
@@ -17,17 +17,24 @@ Sometimes you might want to load you own assets into the playground. Should the
  ```javascript
 myGithubUserName/myRepository/commit/2bd79648e08709145cd9575e6679b2ea360f12f6
````
For me as a beginner a random comment - hash is more self explanatory than something like: 
'myGithubUserName/myRepository/commit/hash'

flopanzer replied:
```suggestion
myGithubUserName/myRepository/commit/hash
```

deltakosh replied:
ok you are right! let's keep it then 
sorry about the change

alexchuber committed:
Refine SmartFilterTexture and input connection documentation

Clarified descriptions for SmartFilterTexture and input connection points in nmeToSfe.md.

bjsplat commented:
snapshot for this PR available at:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/DOCS/refs/pull/1404/merge
